include/onnxruntime/core/platform/threadpool.h

```diff
@@ -54,6 +59,43 @@ class ThreadPool {
   // This is not supported until the latest Eigen
   // void SetStealPartitions(const std::vector<std::pair<unsigned, unsigned>>& partitions);
 
+  /**
+  Tries to call the given function parallelly, splitted into specified batch(es).
+  **/
```

Comment:
Tries to call the given function in parallel, with calls split into batches of the specified size.